### PR TITLE
tfjs implementation

### PR DESCRIPTION
**Pull request by mayaman**
ported the starter code from deeplearnjs to tfjs-core and updated the README where was necessary with the change

### CODE REVIEW

**Comment by mayaman** [comment]
I signed it!

File: package.json [hunk 1]
```diff
@@ -3,9 +3,18 @@
...
   "dependencies": {
-    "deeplearn": "~0.5.0",
-    "deeplearn-knn-image-classifier": "^0.3.0",
-    "pre-commit": "^1.2.2"
+    "deeplearn": "^0.3.12",
+    "deeplearn-knn-image-classifier": "^0.2.0",
+    "@tensorflow-models/knn-classifier": "^0.2.2",
+    "@tensorflow-models/mobilenet": "^0.2.2",
+    "@tensorflow/tfjs": "^0.13.0",
+    "autoless": "^0.1.7",
```
Comment (HalfdanJ) [inline package.json]: Are these new dependencies actually required? Also, deeplearn should be removed

Comment (HalfdanJ) [reply package.json]: Or are you not using budo anymore?

Comment (mayaman) [reply package.json]: still using budo it's still in the dev dependencies... removed deep learn and other unnecessary dependencies

Comment (mayaman) [reply package.json]: okay should be good now... main changes are removed deeplearn, added tfjs, and added babel-polyfill to the dev dependencies

File: main.js [hunk 2]
```diff
@@ -12,8 +12,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import {KNNImageClassifier} from 'deeplearn-knn-image-classifier';
-import * as dl from 'deeplearn';
+require("babel-polyfill");
```
Comment (HalfdanJ) [inline main.js]: Can you change this to `import "@babel/polyfill";`

File: package.json [hunk 3]
```diff
@@ -32,10 +34,12 @@
   },
   "devDependencies": {
     "babel-core": "^6.26.0",
+    "babel-polyfill": "^6.0.16",
```
Comment (HalfdanJ) [inline package.json]: I think its called `@babel/polyfill` now?

Comment (HalfdanJ) [reply package.json]: And can you move this up to dependencies

**Review by HalfdanJ — CHANGES_REQUESTED** [review]

**Review by HalfdanJ — COMMENTED** [review]
Thanks!

**Pull request merged**